src/timeAgo.README.md

```diff
@@ -0,0 +1,38 @@
+
+# `timeAgo`
+
+Takes an absolute date in [epoch format](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#description) and returns a string representing relative time ago.
+
+## Usage
+`timeAgo(epoch, opts)`
+
+Returns: `string | false`
+
+Converts an absolute epoch to a relative time ago string
+
+`epoch`
+
+Type: `number`
+
+The date when an event happened in epoch format
+
+`opts`
+
+Type:
+```typescript
+{
+	verbose?: `boolean`; // Return a longer, more descriptive string when true
+    daysUntilAbsolute?; `number`  // The cutoff for when dates are returned in absolute format
+}
```

Comment:
```suggestion
```typescript
{
	verbose?: boolean, // Return a longer, more descriptive string when true
    daysUntilAbsolute?: number,  // The cutoff for when dates are returned in absolute format
}
```